src/main/java/com/questhelper/helpers/quests/spiritsoftheelid/SpiritsOfTheElid.java

```diff
@@ -133,7 +133,9 @@ protected void setupRequirements()
 		lawRune = new ItemRequirement("Law Rune", ItemID.LAWRUNE, 1);
 		needle = new ItemRequirement("Needle", ItemID.NEEDLE, 1).isNotConsumed();
 		needle.setHighlightInInventory(true);
+		needle.setTooltip("Costume needle cannot be used as a substitute.");
 		thread = new ItemRequirement("Thread", ItemID.THREAD, 2);
+		thread.setTooltip("Costume needle cannot be used as a substitute.")
```

Comment:
oops, one more semicolon missing
```suggestion
		thread.setTooltip("Costume needle cannot be used as a substitute.");
```